oleesir/banking-app
# ft(specific-record):implement specific-record-page

#### What does this PR do?
implement specific-record-template
#### Description of Task to be completed?
implement specific-record-template
#### How should this be manually tested?
* Navigate to the UI folder and click on bank_accounts.html in your browser
* You will see a clickable eye icon
#### What are the relevant pivotal tracker stories?
[#164726911](https://www.pivotaltracker.com/story/show/164726911)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="Screenshot 2019-03-28 at 10 53 25 AM" src="https://user-images.githubusercontent.com/17679511/55148599-ccfae280-5148-11e9-97a9-db3b4e77dcf3.png">
